lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -322,9 +322,8 @@ def create_tar(cv, cvv, repositories, json)
         export_tar = "#{export_prefix}.tar"
         export_dir = File.expand_path(options['option_export_dir'].to_s)
 
-        Dir.mkdir("#{export_dir}/#{export_prefix}")
-
-        if repositories
```

Comment:
You need to check if the content view is not composite here, and completely skip this block if it is.